### ISSUE TITLE
TIMIT using Dropout

### ISSUE THREAD

**Issue by wantee** [yajiemiao/kaldipdnn#1]
We use `run-dnn.sh` in `run_timit/` to test the performance of dropout. With the default setting, we get

> without dropout | learning_rate=0.08
> %WER 19.7 | exp_pdnn/dnn/decode_dev/score_5/ctm_39phn.filt.sys
> %WER 20.8 | exp_pdnn/dnn/decode_test/score_4/ctm_39phn.filt.sys
> 
> with dropout | learning_rate=0.8
> %WER 18.7 | exp_pdnn/dnn/decode_dev/score_4/ctm_39phn.filt.sys
> %WER 20.3 | exp_pdnn/dnn/decode_test/score_4/ctm_39phn.filt.sys

However, if we change the learning rate from 0.08 to 2.048, we get,

> without dropout | learning_rate=2.048
> %WER 18.3 | exp_pdnn/dnn/decode_dev/score_6/ctm_39phn.filt.sys
> %WER 20.1 | exp_pdnn/dnn/decode_test/score_5/ctm_39phn.filt.sys
> 
> with dorpout | learning_rate=20.48
> %WER 18.5 | exp_pdnn/dnn/decode_dev/score_4/ctm_39phn.filt.sys
> %WER 20.1 | exp_pdnn/dnn/decode_test/score_6/ctm_39phn.filt.sys

It can be seen that dropout did not give improvement for the last case. More importantly, by setting learning rate to 2.048 without dropout, we got the best performance, even better than the dropout version in the first case. We also tried other parameter settings, but got no improvement.

Any suggestion to get better performance with dropout?
